publication-rest/src/main/java/no/unit/nva/publication/create/CreatePublicationFromImportCandidateHandler.java

```diff
@@ -0,0 +1,79 @@
+package no.unit.nva.publication.create;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.api.PublicationResponse;
+import no.unit.nva.publication.exception.NotAuthorizedException;
+import no.unit.nva.publication.model.business.ImportCandidate;
+import no.unit.nva.publication.model.business.ImportStatus;
+import no.unit.nva.publication.service.impl.ResourceService;
+import nva.commons.apigateway.AccessRight;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadGatewayException;
+import nva.commons.apigateway.exceptions.NotFoundException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+
+public class CreatePublicationFromImportCandidateHandler extends ApiGatewayHandler<ImportCandidate,
+                                                                                      PublicationResponse> {
+
+    public static final String IMPORT_CANDIDATES_TABLE =
+        new Environment().readEnv("IMPORT_CANDIDATES_TABLE");
+    public static final String ROLLBACK_WENT_WRONG_MESSAGE = "Rollback went wrong";
+    public static final String IMPORT_PROCESS_WENT_WRONG = "Import process went wrong";
+    private final ResourceService importCandidatesService;
```

Comment:
Instead of having two instances of `ResourceService `, maybe it is better to just have all the work in the service it self?

Reply:
Each resourceService is connected to each table, publicationService operates on resource-table and importCandidateService operates on import-candidate-service.

Reply:
I think that sounds like a bad idea. This lambda is working with two different tables. You don't want to accidentally push Import-candidates to NVA resource table or NVA publications into Import-candidate table.

Reply:
Since you have two instances of ResourceService each to a different table, and they are "generic"
maybe `publicationService `and `candidateService` are better names? 
Unless the table you are accessing is called import-candidate, but then I would expect there to be a candidate table as well.

Reply:
What about resourceService and importCandidateService, since one operates on resource-table and second one on import-candidate-service? At the same type Resource is an object we store in database, and both publication and importCandidate are resources at some point.